src/trade/selectors.js

```diff
@@ -0,0 +1,181 @@
+import { createSelector } from 'reselect';
```

Comment:
I think the selectors would be next to the reducers themselves?
I've seen this most often, but we can investigate.